app/hooks/use-price-conversion.ts

```diff
@@ -1,10 +1,21 @@
-import { WalletCurrency } from "@app/graphql/generated"
-import { usePriceContext } from "@app/store/price-context"
+import {
+  CurrentPriceDocument,
+  WalletCurrency,
+  useCurrentPriceQuery,
+} from "@app/graphql/generated"
 import { PaymentAmount, UsdPaymentAmount } from "@app/types/amounts"
 import * as React from "react"
 
 export const usePriceConversion = () => {
-  const priceData = usePriceContext()
+  const { data } = useCurrentPriceQuery({
+    query: CurrentPriceDocument,
+    fetchPolicy: "cache-only",
```

Comment:
Apollo recommends using `fetchPolicy: cache-and-network` for data that frequently updates. Maybe we can try that. https://www.apollographql.com/docs/react/data/queries/#cache-and-network

Reply:
we already get the update from the subscription.